charts/sourcegraph/templates/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -45,6 +45,26 @@ spec:
         app: sourcegraph-frontend
         deploy: sourcegraph
     spec:
+      initContainers:
+      {{- if .Values.migrator.enabled }}
+      - name: migrator
+        image: {{ include "sourcegraph.image" (list . "migrator") }}
+        args:
+        - up
```

Comment:
This should probably be a configurable option

Reply:
> This should probably be a configurable option

I was hoping we can make all args or command configurable at once via https://github.com/sourcegraph/sourcegraph/issues/29063

Reply:
😆  I forgot that issue existed / hadn't been completed yet